resources/testharness.js

```diff
@@ -4254,18 +4257,6 @@
                              });
                 });
 
-        // This use of innerHTML plus manual escaping is not recommended in
-        // general, but is necessary here for performance.  Using textContent
-        // on each individual <td> adds tens of seconds of execution time for
-        // large test suites (tens of thousands of tests).
```

Comment:
I assume you've tested this? Please explain in the PR summary

Reply:
On my machine; current version of Chrome; on the largest result table that we found (html/dom/reflection-text.text; with indeed >10k entries) the results generation took ~800ms in total. So the total time taken by the `textContent` calls would be clearly less than "tens of seconds".

Reply:
Could you also compare in Firefox and Safari?

Reply:
On Chrome, I measured (with the latest version) also about ~800ms for the results generation, vs. ca 650ms for the original code.
On Firefox, I get about 1100ms, vs ~800ms previously.
I don't have a machine that Safari runs on.

I'll note that both of those measurements have very high variance, and on some runs the old code was just as slow as the new one; but usually it seemed a little faster. I'm not seeing the "tens of seconds of execution time" that the comment promises, anywhere.

How is this usually measured? Is there a way to get performance data out of the bot runs that are done for each PR?